lib/elixir/lib/task.ex

```diff
@@ -341,8 +341,16 @@ defmodule Task do
   Each `enumerable` item is passed as argument to the given function `fun` and
   processed by its own task. The tasks will be linked to the current process,
   similarly to `async/1`.
-
-  See `async_stream/5` for discussion, options, and examples.
+  
+  ## Example
+  
+  Count the graphemes in each string asynchronously, then add the counts together using reduce.
+  
+      iex> stream = Task.async_stream(["long string", "longer string", "there are many of these"], fn text -> text |> String.graphemes |> Enum.count end)
```

Comment:
Let's move the first argument to a variable so everything fits neatly in a single line:

```iex
iex> list = ["long string", "longer string", "there are many of these"]
iex> stream = Task.async_stream(stream, fn text -> text |> String.graphemes |> Enum.count end)
```